exercises/raindrops/canonical-data.json

```diff
@@ -1,74 +1,92 @@
 {
    "cases": [
       {
+         "description" : "the sound for 1 is 1",
          "number" : 1,
          "expected": "1"
       },
       {
+         "description" : "the sound for 3 is Pling",
          "number" : 3,
          "expected": "Pling"
       },
       {
+         "description" : "the sound for 5 is Plang",
          "number" : 5,
          "expected": "Plang"
       },
       {
+         "description" : "the sound for 7 is Plong",
          "number" : 7,
          "expected": "Plong"
       },
       {
+         "description" : "the sound for 6 is Pling as it has a factor 3",
          "number" : 6,
          "expected": "Pling"
       },
       {
+        "description" : "the sound for 8 is 8",
```

Comment:
That proposed description seems good.

Caveat on the strength of my opinion: Whether it qualifies for common isn't known first-hand to me as I haven't done this exercise in any language. A particular way of doing it in Ruby (https://github.com/exercism/xruby/issues/434) first caused this case to be added.

But I figure... now that it's added, might as well give it the description to suit it, and I think this description does suit.